lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM/SharedToDotOperandMMAv2.cpp

```diff
@@ -110,8 +112,8 @@ MMA16816SmemLoader::computeLdmatrixMatOffs(Value warpId, Value lane,
   Value s1 = udiv(matIndex, i32_val(2));
 
   // We use different orders for a and b for better performance.
-  Value kMatArr = kOrder == 1 ? s1 : s0;  // index of matrix on the k dim
-  Value nkMatArr = kOrder == 1 ? s0 : s1; // index of matrix on the non-k dim
+  Value kMatArr = kOrder == 2 ? s1 : s0;  // index of matrix on the k dim
```

Comment:
nit: might be worth making some variable with names such as `b, m, n` or something like that at some point to improve readability

Reply:
Do you mean for `s0` and `s1` ?

Reply:
I meant for all the `0`, `1`, `2` kind of numbers